src/boot/efi/stub.c

```diff
@@ -88,6 +97,7 @@ static void export_variables(EFI_LOADED_IMAGE_PROTOCOL *loaded_image) {
                 EFI_STUB_FEATURE_PICK_UP_SYSEXTS |          /* We pick up system extensions from the boot partition */
                 EFI_STUB_FEATURE_THREE_PCRS |               /* We can measure kernel image, parameters and sysext */
                 EFI_STUB_FEATURE_RANDOM_SEED |              /* We pass a random seed to the kernel */
+                EFI_STUB_FEATURE_PICK_UP_ADDONS |           /* We load addons provided via EFI */
```

Comment:
i think we should advertise which sections precisely we pick up via addons, i.e. distinct flags for this. as we add support for more userspace wants to know which ones

Reply:
Split into 2 pick up features, 1 feature for sd-boot called "offer addons"